app/views/maps/_map.html.erb

```diff
@@ -4,6 +4,7 @@
     action: "click->campaign#chooseMapOption",
     map_id: map.id
   } do %>
+    <%= link_to "Edit", edit_map_path(map), remote: true %>
```

Comment:
How do you feel about the look of the edit link? I'm wondering about hiding it until the map is hidden or putting it in a corner?

<img width="372" alt="Screenshot 2020-05-31 at 09 56 57" src="https://user-images.githubusercontent.com/1191840/83348487-133b6b00-a325-11ea-9147-65569cc5c61b.png">